src/client/theme-default/index.ts

```diff
@@ -6,10 +6,21 @@ import './styles/custom-blocks.css'
 import Layout from './Layout.vue'
 import NotFound from './NotFound.vue'
 import { Theme } from '../app/theme'
+import { installGoogleAnalytics } from './ga'
 
 const theme: Theme = {
   Layout,
-  NotFound
+  NotFound,
+  enhanceApp({ router, siteData }) {
+    const { googleAnalytics: GA_ID } = siteData.value.themeConfig
+    if (
+      GA_ID &&
+      process.env.NODE_ENV !== 'production' &&
```

Comment:
Did you mean `process.env.NODE_ENV === 'production'`?

Reply:
Yes, thanks